app/services/hooks/useAssets.js

```diff
@@ -1,14 +1,15 @@
 import { useTranslation } from 'react-i18next';
-import { Images } from './assets';
-import { isGPS } from './COVIDSafePathsConfig';
+
+import { Images } from '../../assets';
```

Comment:
Im less of a fan of grouping by implementation and more of a fan of grouping by feature. And in general, I try to keep relative paths shorter as a metric to determine approachability to the code base.

What do you think about not moving this file?